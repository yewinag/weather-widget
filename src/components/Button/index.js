import styled from 'styled-components';

const Button = styled.button`
    padding: 10px;
    color: #fff;
    background: #0A56BB;
    border:0;
    border-radius: 4px;
    padding-left: 16px;
    padding-right: 16px;
`;
export default Button;