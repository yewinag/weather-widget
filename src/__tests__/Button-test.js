import React from 'react';

import renderer from 'react-test-renderer';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

import Button from '../components/Button';

Enzyme.configure({ adapter: new Adapter() });

describe('button', () => {
    it('render button', () => {
        const wrapper = shallow(<Button />)
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });
});