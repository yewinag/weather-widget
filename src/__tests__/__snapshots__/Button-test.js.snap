// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button render button 1`] = `ShallowWrapper {}`;
